ggshield/config.py

```diff
@@ -153,8 +151,32 @@ class UserConfig(YamlFileConfig):
     banlisted_detectors: Set[str] = field(default_factory=set)
     ignore_default_excludes: bool = False
 
-    def __post_init__(self) -> None:
-        self.api_url = os.getenv("GITGUARDIAN_API_URL", DEFAULT_API_URL)
+    def set_env_gitguardian_urls(self) -> None:
```

Comment:
(naming) I thought this was updating environment variables. I would rename it to something like `update_from_env()`.